docs/pages/faq/z-index.md

```diff
@@ -0,0 +1,253 @@
+        <View style={[styles.item, { backgroundColor: '#4af2a1' }]} />
+      </View>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  root: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  container: {
+    height: 300,
+    width: 300,
+    borderRadius: 16,
+    padding: 16,
+    borderWidth: 8,
+    borderColor: 'rgba(0,0,0,0.2)',
+  },
+  item: {
+    borderWidth: 4,
+    borderColor: 'rgba(0,0,0,0.2)',
+    height: 64,
+    width: 64,
+    borderRadius: 8,
+  },
+});
+```
+
+This is illustrated more clearly when the components visually intersect each other.
+
+<ImageSpotlight style={{maxWidth: 360}} alt="Three square components intersecting each other" src="/static/images/z-index/default-visually-stacked.png" />
+
+```jsx
+export default function App() {
+  return (
+    <View style={styles.root}>
+      <View
+        style={[
+          styles.container,
+          {
+            backgroundColor: '#e1e4e8',
+          },
+        ]}>
+        {/* zIndex: 0 */}
+        <View style={[styles.item, { backgroundColor: '#6638f0' }]} />
+        {/* zIndex: 1 */}
+        <View style={[styles.item, { backgroundColor: '#5cc9f5', marginTop: -16 }]} />
+        {/* zIndex: 2 */}
+        <View style={[styles.item, { backgroundColor: '#4af2a1', marginTop: -16 }]} />
+      </View>
+    </View>
+  );
+}
+```
+
+## Changing the `zIndex` of an element
+
+If you want to change how a component stacks without changing the order that it occurs in the component tree, use `zIndex`:
+
+<ImageSpotlight style={{maxWidth: 360}} alt="Three components where the second is stacked above the first and third" src="/static/images/z-index/relative-z-index.png" />
+
+```jsx
+export default function App() {
+  return (
+    <View style={styles.root}>
+      <View
+        style={[
+          styles.container,
+          {
+            backgroundColor: '#e1e4e8',
+          },
+        ]}>
+        <View style={[styles.item, { backgroundColor: '#6638f0' }]} />
+        <View style={[styles.item, { backgroundColor: '#5cc9f5', marginTop: -16, zIndex: 1 }]} />
+        <View style={[styles.item, { backgroundColor: '#4af2a1', marginTop: -16 }]} />
+      </View>
+    </View>
+  );
+}
+```
+
+## Manually positioning your component
+
+Along with specifying how the component will stack, you can break out of the default layout set by the component's parent and by changing the `position` property on that component to `'absolute'` and specifying the distance it should be from its parent with the style properties `top`, `right`, `bottom`, and `left`.
+
+<ImageSpotlight style={{maxWidth: 360}} alt="Position absolute example" src="/static/images/z-index/absolute-position.png" />
+
+```jsx
+export default function App() {
+  return (
+    <View style={styles.root}>
+      <View
+        style={[
+          styles.container,
+          {
+            backgroundColor: '#e1e4e8',
+          },
+        ]}>
```

Comment:
Keep the page concise, especially since this part of the example isn't the point to highlight
```suggestion
        style={[styles.container, { backgroundColor: '#e1e4e8' }]}>
```
(same recommendation throughout this page -- I didn't leave explicit comments elsewhere)